addons/danger/functions/fnc_brainEngage.sqf

```diff
@@ -50,6 +50,13 @@ if (
     && {(vehicle _target) isKindOf "CAManBase"}
     // && {_target call EFUNC(main,isAlive)}
 ) exitWith {
+
+    // set incapacitated ACE units visible
+    if (!GVAR(disableIncapacitatedKills)) then {
+        private _vis = _target getVariable ["ace_common_oldVisibility", 1];
```

Comment:
Are you sure `_target` is local here? `ace_common_oldVisibility` is only available where `_target` is local, `setUnitTrait` also requires the `_target` to be local and will globally propagate the camouflage change.